src/amp-story-player/amp-story-player-iframe-pool.js

```diff
@@ -77,4 +77,58 @@ export class IframePool {
 
     return detachedStoryIdx;
   }
+
+  /**
+   * Clears stored story indices.
+   * @return {!Array<number>}
+   */
+  evictStories() {
+    const evictedStories = this.storyIdsWithIframe_;
+    this.storyIdsWithIframe_ = [];
+    return evictedStories;
+  }
+
+  /**
+   * @return {!Array<number>}
+   */
+  getAvailableIframeIdx() {
+    return this.iframePool_;
+  }
+
+  /**
+   * Finds adjacent iframe indices given an index.
+   * Examples (x is provided index):
+   * [x] [1] [2] [] []
```

Comment:
I'm not sure I understand the diagram here; is this a representation of `queue` or `adjacent`?

Reply:
It's a representation of the resulting `adjacent` array given a provided `x` index. I'll update the JSDoc to see if I can clear it up.